joomla/joomla-cms
# [5.0] Update deleted files list in script.php to recent changes in the 5.0-dev branch

Pull Request for Issue # .

### Summary of Changes

This pull request (PR) updates the lists of files and folders to be deleted on update in file "administrator/components/com_admin/script.php" to recent changes in the 5.0-dev branch.
- New 4.x update SQL scripts are added to the deleted files list.
- File '/libraries/vendor/google/recaptcha/ARCHITECTURE.md' is added to the deleted files list due to a dependency update.
- Files and folders of symfony polyfills for unsupported PHP versions are added to the lists.
- Files and folders from the symfony/console dependency are removed from the lists as they have been added back to the package by a dependency update.

### Testing Instructions

Code review.

Or if you want to make a real test, update the last 4.4 nightly build to the last 5.0 nightly build to get the actual result, and update the last 4.4 nightly build to the update package built by Drone for this PR to get the expected result. Then compare the result with the result of a new installation with the full installation package.

### Actual result BEFORE applying this Pull Request

The files and folders mentioned above as added to the lists are still present after updating from the last 4.4 nightly build.

Files and folders of the symfony/console dependency which are present in the full installation package are deleted with the update.

### Expected result AFTER applying this Pull Request

The files and folders mentioned above as added to the lists have been deleted after updating from the last 4.4 nightly build.

Files and folders of the symfony/console dependency which are present in the full installation package are also present after the the update.

### Link to documentations
Please select:
